plugins/pulseaudio/PulseAudioDevice.cpp

```diff
@@ -139,6 +150,7 @@ void PulseAudioDevice::playing(bool playing)
 	{
 		AUD_pa_threaded_mainloop_lock(m_mainloop);
 		AUD_pa_stream_cork(m_stream, 0, nullptr, nullptr);
+		m_synchronizer.update_time_start();
```

Comment:
This makes the synchronizer timing depended on when you play back sounds. That's not right. You should implement `ISynchronizer::play()` and set the start time there.